lib/promoted/ruby/client/request_builder.rb

```diff
@@ -100,35 +100,32 @@ def fill_details_from_response response_insertions
           filled_in_copy
         end
 
-        def log_request_params(include_insertions: true, include_request: true)
+        def log_request_params(include_delivery_log: true, exec_server: Promoted::Ruby::Client::EXECUTION_SERVER['SDK'])
```

Comment:
It might make sense to not have default values and force callers to set the values.

Reply:
Ok. We haven't really been using the named-params-w/o-defaults language feature, but can start.